src/Corax/Analyzers/Analyzer.cs

```diff
@@ -11,6 +11,7 @@ namespace Corax
 {
     public unsafe class Analyzer : IDisposable
     {
+        public static Analyzer DefaultAnalyzer = Create(default(KeywordTokenizer), default(ExactTransformer));
```

Comment:
Shouldn't this be the lower case one? That is what we typically consider the default in RavenDB

Reply:
Maybe we can just create that instance in the test only?

Reply:
That should match the RavenDB one. That is correct.